projects/kubernetes/cloud-provider-aws/expected_artifacts

```diff
@@ -4,10 +4,10 @@ SHA256SUM.sha512
 SHA512SUM
 SHA512SUM.sha256
 SHA512SUM.sha512
-cloud-provider-aws-linux-amd64-v1.26.1.tar.gz
-cloud-provider-aws-linux-amd64-v1.26.1.tar.gz.sha256
-cloud-provider-aws-linux-amd64-v1.26.1.tar.gz.sha512
-cloud-provider-aws-linux-arm64-v1.26.1.tar.gz
-cloud-provider-aws-linux-arm64-v1.26.1.tar.gz.sha256
-cloud-provider-aws-linux-arm64-v1.26.1.tar.gz.sha512
+cloud-provider-aws-linux-amd64-v1.27.0.tar.gz
```

Comment:
You shouldn't need the version in this file. Check one of the others I believe you can use GIT_TAG

Reply:
done